server/migrations/20210925190654-create-user.ts

```diff
@@ -0,0 +1,46 @@
+"use strict";
+module.exports = {
+  up: async (queryInterface: any, Sequelize: any) => {
+    await queryInterface.createTable("Users", {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.UUIDV4,
+        unique: true
+      },
+      userName: {
+        type: Sequelize.STRING(8),
+        allowNull: false,
+        unique: true
+      },
+      password: {
+        type: Sequelize.STRING,
+        allowNUll: false
+      },
+      salt: {
+        type: Sequelize.STRING,
+      },
+      email: {
+        type: Sequelize.STRING,
+        unique: true
+      },
+      lastLogin: {
+        type: Sequelize.DATE,
```

Comment:
Can we add a hook here to automatically update this after an update happens?

Reply:
I think this can be done after the login feature is added. I'm not sure how to add a hook without having defined what it means to be logged in.